GalaxyZooTouchTable/GalaxyZooTouchTable/UserConsole.xaml.cs

```diff
@@ -10,19 +14,45 @@ namespace GalaxyZooTouchTable
     public partial class UserConsole : UserControl
     {
         ClassificationPanel Classifier { get; set; }
+        public bool ClassifierOpen { get => classifierOpen; set => classifierOpen = value; }
```

Comment:
Couple of things here:
1) Why do you have a public and private version of this property? Seems like it would just be fine if you left this as a `public bool` and got rid of the `private` one.

2) Not a huge deal, but I think the name of this property is kind of confusing. `ClassifierOpen` sounds like an action rather than a property. It may be more clear to name this something like `IsClassifierOpen` or `ClassifierIsOpen`.

Reply:
Are you going to make these changes?

Reply:
Yes. Will comment below.